R/package.r

```diff
@@ -89,7 +89,9 @@ package_rd <- function(path) {
 extract_tag <- function(x, tag) {
   x %>%
     purrr::keep(inherits, tag) %>%
-    purrr::map_chr(c(1, 1))
+    unlist %>%
+    paste(collapse = " ") %>%
+    stringr::str_trim()
```

Comment:
Ah, I did not know base::trimws exists. 

You are right, the approach fails e.g. when there is more than one \alias tag in the Rd file. So somehow we need to differentiate between lists with several elements originating from a multiline tag, (e.g. \title) and lists coming from several tags (e.g. several occurrences of \alias).